prometheus-compliance-tests failure

The `prometheus-compliance-tests` step is failing with the following reported error:

```
Run mkdir compliance/remote_write/bin && cp ./bin/otelcontribcol_linux_amd64 compliance/remote_write/bin/otelcol_linux_amd64
mkdir: cannot create directory ‘compliance/remote_write/bin’: No such file or directory
Error: Process completed with exit code 1.
```

See https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/4071763859?check_suite_focus=true

--- a/.github/workflows/prometheus-compliance-tests.yml
+++ b/.github/workflows/prometheus-compliance-tests.yml
@@ -44,8 +44,9 @@ jobs:
         with:
           repository: prometheus/compliance
           path: compliance
+          ref: bc7dd4cb6e88e60b702023c199140281c4dce906
       - name: Copy binary to compliance directory
-        run: mkdir compliance/remote_write/bin && cp ./bin/otelcontribcol_linux_amd64 compliance/remote_write/bin/otelcol_linux_amd64
+        run: mkdir compliance/remote_write_sender/bin && cp ./bin/otelcontribcol_linux_amd64 compliance/remote_write_sender/bin/otelcol_linux_amd64
       - name: Run compliance tests
         run: go test --tags=compliance -run "TestRemoteWrite/otel/.+" -v ./
-        working-directory: compliance/remote_write
+        working-directory: compliance/remote_write_sender

